src/com/reco1l/osu/ui/SettingsFragment.kt

```diff
@@ -0,0 +1,392 @@
+package com.reco1l.osu.ui
+
+import android.annotation.SuppressLint
+import android.content.ClipData
+import android.content.ClipboardManager
+import android.content.Intent
+import android.graphics.Color
+import android.net.Uri
+import android.os.Bundle
+import android.text.InputType.TYPE_CLASS_TEXT
+import android.text.InputType.TYPE_TEXT_VARIATION_PASSWORD
+import android.view.ContextThemeWrapper
+import android.view.View
+import android.widget.LinearLayout
+import android.widget.TextView
+import androidx.annotation.XmlRes
+import androidx.core.content.getSystemService
+import androidx.core.view.forEach
+import androidx.core.view.get
+import androidx.preference.CheckBoxPreference
+import androidx.preference.EditTextPreference
+import androidx.preference.ListPreference
+import androidx.preference.Preference
+import com.edlplan.ui.SkinPathPreference
+import com.edlplan.ui.fragment.LoadingFragment
+import com.google.android.material.snackbar.Snackbar
+import com.reco1l.ibancho.LobbyAPI
+import com.reco1l.ibancho.RoomAPI
+import com.reco1l.ibancho.data.RoomTeam
+import com.reco1l.ibancho.data.TeamMode
+import com.reco1l.ibancho.data.WinCondition
+import com.reco1l.osu.UpdateManager
+import com.reco1l.osu.async
+import com.reco1l.osu.mainThread
+import com.reco1l.osu.multiplayer.Multiplayer
+import com.reco1l.osu.multiplayer.RoomScene
+import com.reco1l.toolkt.android.bottomMargin
+import com.reco1l.toolkt.android.dp
+import com.reco1l.toolkt.android.drawableLeft
+import com.reco1l.toolkt.android.layoutWidth
+import com.reco1l.toolkt.android.topMargin
+import ru.nsu.ccfit.zuev.osu.Config
+import ru.nsu.ccfit.zuev.osu.GlobalManager
+import ru.nsu.ccfit.zuev.osu.LibraryManager
+import ru.nsu.ccfit.zuev.osu.MainActivity
+import ru.nsu.ccfit.zuev.osu.PropertiesLibrary
+import ru.nsu.ccfit.zuev.osu.ResourceManager
+import ru.nsu.ccfit.zuev.osu.ToastLogger
+import ru.nsu.ccfit.zuev.osu.helper.StringTable
+import ru.nsu.ccfit.zuev.osu.online.OnlineManager
+import ru.nsu.ccfit.zuev.osuplus.R
+import ru.nsu.ccfit.zuev.skins.SkinManager
+import java.io.File
+
+
+enum class Section(@XmlRes val xml: Int) {
+
+    General(R.xml.settings_general),
+    Gameplay(R.xml.settings_gameplay),
+    Graphics(R.xml.settings_graphics),
+    Sounds(R.xml.settings_audio),
```

Comment:
```suggestion
    Audio(R.xml.settings_audio),
```

I think calling this `Audio` is more suitable.